code/game/objects/structures/crates_lockers/closets/secure/security_lockers.dm

```diff
@@ -119,7 +119,14 @@
 	icon_opened = "hop_open"
 
 /obj/structure/closet/secure_closet/blueshield/populate_contents()
+	if(prob(50))
+		new /obj/item/storage/backpack/blueshield(src)
+	else
+		new /obj/item/storage/backpack/satchel_blueshield(src)
```

Comment:
Why a 50/50 and not just add both to the locker?

Reply:
Sure, why not.